integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanTransactionChargebackTest.java

```diff
@@ -114,7 +116,7 @@ public void applyLoanTransactionChargeback() {
 
         // Try to reverse a Loan Transaction charge back
         PostLoansLoanIdTransactionsResponse reverseTransactionResponse = loanTransactionHelper.reverseLoanTransaction(loanId,
-                chargebackTransactionId, operationDate, responseSpecErr503);
+                chargebackTransactionId, operationDate, responseSpecErr403);
```

Comment:
So reversing or adjusting a chargeback is not possible, right?

Reply:
Yes, I needed to change the HTTP code due I sync the Exception used in this case to `InvalidLoanTransactionTypeException`